tidy up the readme

nerdalert opened the pull request:
Just cleaned up the snippets to make the markdown consistant.

Fan of the project. Cheers!

fentas commented:
LGTM

fentas commented on README.md:
````diff
@@ -162,24 +182,29 @@ Run `make deploy`
     
     sudo -E easy_install appscript
-    
+```
````
Could you remove whitespaces here? Or do they have a purpose?

nerdalert commented:
TY for review! Didn't want to be too presumptuous on the whitespaces :) Removed the line breaks in the snippets, removed a few rando tabs on empty lines and added shell syntax! 
Cheers!
